File: M8/Exemplos/App_eXperience_Java/app/src/main/java/com/elgin/elginexperience/shipay/models/BuyerInfo.java
package com.elgin.elginexperience.shipay.models;

import com.google.gson.annotations.SerializedName;

public class BuyerInfo {

    @SerializedName("address")
    public String address;

    @SerializedName("city")
    public String city;

    @SerializedName("document")
    public String document;

    @SerializedName("email")
    public String email;

    @SerializedName("first_name")
    public String first_name;

    @SerializedName("last_name")
    public String last_name;

    @SerializedName("phone")
    public String phone;

    @SerializedName("state")
    public String state;
}

File: Exemplos/App_eXperience_Kotlin/Elgin_AppExperience_Kotlin-font/app/src/main/java/com/example/e1_kotlin_r/shipay/models/BuyerInfo.kt
package com.example.e1_elgin_kotlin.shipay.models

import com.google.gson.annotations.SerializedName

data class BuyerInfo (
    @SerializedName("address")
    var address: String,

    @SerializedName("city")
    var city: String,

    @SerializedName("document")
    var document: String,

    @SerializedName("email")
    var email: String,

    @SerializedName("first_name")
    var first_name: String,

    @SerializedName("last_name")
    var last_name: String,

    @SerializedName("phone")
    var phone: String,

    @SerializedName("state")
    var state: String
)
